### ISSUE TITLE
Dragging the title bar from the icon or title doesn't move the window (Windows)

### ISSUE THREAD

**Issue by SeparateRecords** [onivim/oni2#2449]
Potentially low-hanging fruit and low impact. I'm interested in fixing this one myself to get a little bit familiar with the project internally. Looks like you'd just need to bubble up the drag/click event from the title bar's children to the element with the event handler?

Had a quick skim through and couldn't immediately see anything relevant to this, but I'm probably missing something :)

**Comment by bryphe** [comment]
Thanks for logging the issue, @SeparateRecords !

>  I'm interested in fixing this one myself to get a little bit familiar with the project internally. Looks like you'd just need to bubble up the drag/click event from the title bar's children to the element with the event handler?

Ah, sorry I grabbed this from you - I opened a PR in #2457. Fix was essentially as you suggested - we have a special attribute `mouseBehavior` to mark elements that can be used to drag the window: 

```
      <View
        mouseBehavior=Draggable
        ...
     />
```

However, the `<Text />` and `<Image />` components inside were preferentially getting the mouse event. I added an attribute to have both those elements ignore mouse events, while lets that `mouseBehavior` get at it again.
